src/Kupo/App/Http.hs

```diff
@@ -164,6 +167,9 @@ app withDatabase forceRollback patternsVar readHealth req send =
         ("patterns" : args) ->
             routePatterns (requestMethod req, args)
 
+        ("outputs" : args) ->
+            routeOutputs (requestMethod req, args)
```

Comment:
On a second thought, this doesn't feel quite right. Because the handler here still returns `Result` (or so-called `matches` in the API terminology). Thus, the resource being accessed here are `matches` and not `outputs`. 

So, I'd suggest to amend patterns to also capture output references as well, including wildcards. I am thinking about something like:

```
            ┏━━━━━━━━━┓
PATTERN  ╾┬─┫ ADDRESS ┣───────────────────────────────┬╼
          │ ┗━━━━━━━━━┛                               │
          │ ┏━━━━━━━━━━━━━━━┓                         │
          ├─┫ STAKE-ADDRESS ┣─────────────────────────┤
          │ ┗━━━━━━━━━━━━━━━┛                         │
          │ ┏━━━━━━━━━━━━┓ ╭───╮ ┏━━━━━━━━━━━━┓       │
          ├─┫ CREDENTIAL ┣─┤ / ├─┫ CREDENTIAL ┣───────┤
          │ ┗━━━━━━━━━━━━┛ ╰───╯ ┗━━━━━━━━━━━━┛       │
          │ ┏━━━━━━━━━━━━━━┓ ╭───╮ ┏━━━━━━━━━━━━━━━━┓ │
          ├─┫ OUTPUT-INDEX ┣─┤ @ ├─┫ TRANSACTION-ID ┣─┤
          │ ┗━━━━━━━━━━━━━━┛ ╰───╯ ┗━━━━━━━━━━━━━━━━┛ │
          │ ╭───╮                                     │
          └─┤ * ├─────────────────────────────────────┘
            ╰───╯

                ┏━━━━━━━━━━━┓
OUTPUT-INDEX ╾┬─┫ 1*3 DIGIT ┣─┬╼
              │ ┗━━━━━━━━━━━┛ │
              │ ╭───╮         │
              └─┤ * ├─────────┘
                ╰───╯

                  ┏━━━━━━━━━━━━━━━━━━━━━━━━┓
TRANSACTION-ID ╾┬─┫ BASE16(bytes .size 32) ┣─┬╼
                │ ┗━━━━━━━━━━━━━━━━━━━━━━━━┛ │
                │ ╭───╮                      │
                └─┤ * ├──────────────────────┘
                  ╰───╯
```

with `CREDENTIAL` remaining unchanged. 

Not sure about the wildcard for `TRANSACTION-ID` though.. I don't think it makes much sense to search for all UTxO that were the n-th output of a transaction :thinking: ? 

With this, we can then simply hook it in the `/matches` endpoint, and have clients application look for e.g. `*@f13014f84c32a9f445627e690e8b908c612dbdc6d3970051e59daf8d18ae878b`